### DISCUSSION TITLE
Enabling CORS in Oxigraph

### DISCUSSION THREAD

**Discussion by mikel-egana-aranguren** [oxigraph/oxigraph#806] [Q&A] [answered]
Hi;

I'm trying to query data stored in Oxigraph from an external web page (with embeded YASGUI), but it looks like I need to enable CORS in Oxigraph. What is the best way of doing so? I haven´t found anything in the documentation. I'm deploying Oxigraph through the Docker image (ghcr.io/oxigraph/oxigraph:v0.3.22) and I can query the data with the YASGUI included in Oxigraph (http://localhost:7878)

Thank you

**Comment by Tpt** [comment, marked as answer]
Hey! Oxigraph server is not supposed to be used directly but behind a reverse proxy when exposed publicly. It's why it does not support CORS headers and TLS termination.

In the README there is [an example with nginx and auth](https://github.com/oxigraph/oxigraph/tree/main/cli#run-the-web-server-with-basic-authentication), CORS should not be too hard to add

**Reply by mikel-egana-aranguren** [reply]
Understood, thank you.
